.tool-versions

```diff
@@ -0,0 +1 @@
+pnpm 8.15.5
```

Comment:
is this meant to be committed or for your local env?

Reply:
it's for the repo. it'll help contributors make sure they use pnpm 8.x which our lock file requires

Reply:
if helpful, ankur had a similar question: https://github.com/braintrustdata/braintrust-proxy/pull/128/files#r1898113070